I added the activation processing of the SWAP partition here because the mounting of the target device should be complete at the point of the unpackfs module

phoepsilonix opened the pull request:
…lso be added to automatically mount the swap partition if it exists when installing the device, including if manual partitioning is performed. This is because, especially in kernel-6.1, in processes with a large memory usage such as copying a large amount of files or generating initramfs, the OOM Killer will force the process to stop not only the installer but also the desktop environment, causing the installation to be interrupted. If there is no SWAP partition, it may also be possible to create a swapfile on the target device being installed. I think such processing should be added somewhere, but for now, I added the activation processing of the SWAP partition here because the mounting of the target device should be complete at the point of the unpackfs module

dalto8 reviewed (changes requested):
As mentioned in the last PR, working with swap doesn't belong in unpackfs.  Many installations don't even use unpackfs so this is will never get called.  Even if they did, this is not the right location to activate swap.

This probably should happen in the mount module(as an option in the config) or in it's own module.

This also has all the same code quality issues as noted in detail in the prior PR.  I described them in detail over there but a quick summary would be:
* Use calamares process calls instead of `subprocess.call`.  These are described in modules readme.
* Replace `sh -c` and shell pipelining with Python logic
* Don't use `sudo`
* Ensure you have proper exception handling anywhere an exception can be thrown

phoepsilonix committed:
During installation, if a swap partition exists, enable swap.

If there is no swap at all when the partition of the installation destination device is mounted, if a Swap partition exists, enable swap.
target_env_call cannot be used because it is before the files are copied to the installation destination.
host_env_process_output cannot be used because the return value cannot be obtained.
To avoid reinventing the wheel as much as possible, use subprocess.check_output.
